include/TaskListDlg.h

```diff
@@ -54,7 +55,22 @@ public :
 			return;
 		//clear list LB_RESETCONTENT
 		::SendMessage( _hList, LB_RESETCONTENT, NULL, NULL );
+		if ( !isDarkMode() && !todoItems.empty() )

```

Comment:
It seems to be sufficient to add

`
RedrawWindow(_hParent, NULL, NULL, RDW_ERASE | RDW_FRAME | RDW_INVALIDATE | RDW_ALLCHILDREN);`

to get the drawing done. Just the background is still not correctly done.